src/test/scala/chiselTests/ToTargetSpec.scala

```diff
@@ -183,4 +214,17 @@ class ToTargetSpec extends ChiselFlatSpec with Utils {
 
     (e.getMessage should include).regex("Requested .toRelativeTarget relative to .+, but it is not an ancestor")
   }
+
+  it should "work to get relative targets to an instance of an Instance" in {
+    val chirrtl = ChiselStage.emitCHIRRTL(new RelativeSiblingsInstancesParent)
+
+    chirrtl should include("~RelativeSiblingsInstancesParent|Foo>inner")
```

Comment:
not the right check yet, but not even getting to the point of chirrtl emission yet